README.md

```diff
@@ -234,6 +234,59 @@ func connect() {
     // ... etc
 }
 ```
+### Using DNS to identify an instance
+
+The connector can be configured to use DNS to look up an instance. This would
+allow you to configure your application to connect to a database instance, and
+centrally configure which instance in your DNS zone.
+
+#### Configure your DNS Records
+
+Add a DNS SRV record for the Cloud SQL instance to the DNS used by your 
```

Comment:
People will do this for public IP. We should warn them that it leaks PII if they do that. This should be recommended for private contexts only.